alpha/declcfg/load_test.go

```diff
@@ -337,6 +337,170 @@ func TestLoadFS(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:      "Success/ValidDir with deprecations",
```

Comment:
Is there a particular reason to make this a separate test case completely as opposed to adding the deprecations to the existing test case named `Success/ValidDir`?

Reply:
[e9afb84b36b44e76b20defac5b73747afd7fdd46](https://github.com/operator-framework/operator-registry/pull/1151/commits/e9afb84b36b44e76b20defac5b73747afd7fdd46)